x-pack/legacy/plugins/ml/public/application/util/string_utils.js

```diff
@@ -309,7 +309,7 @@ export function guessTimeFormat(time) {
 export function toLocaleString(x) {
   let result = x;
   if (x && typeof x === 'number') {
-    const parts = x.toString().split('.');
```

Comment:
do we want to be round to 3 decimal places for all uses of this function?
maybe instead the value should be run through `roundToDecimalPlace` before being passed to `toLocaleString `

Reply:
Check https://github.com/elastic/kibana/pull/55062/commits/28ed6dc83bd3cd1b8014ab2ffa59e4c280edfaeb